pmm-app/src/pmm-check/CheckPanel.test.tsx

```diff
@@ -58,4 +58,27 @@ describe('CheckPanel::', () => {
     spy.mockClear();
     wrapper.unmount();
   });
+
+  it('should render the link to Settings when STT is disabled', async () => {
+    const spy = jest.spyOn(CheckService, 'getSettings').mockImplementation(() => Promise.resolve({
+      settings: {
+        stt_enabled: false,
+      },
+    }));
+
+    const wrapper: ReactWrapper<{}, {}, any> = mount(<CheckPanelRouter />);
+
+    await runAllPromises();
+    wrapper.update();
+
+    expect(wrapper.find(dataQa('db-check-panel-settings-link'))).toHaveLength(1);
+    const text = 'Security Threat Tool is disabled. You can enable it in PMM Settings.';
```

Comment:
Probably this should be stored in Messages and used both here and in the component

Reply:
Done [in this PR](https://github.com/percona/grafana-dashboards/pull/1109)